### ISSUE TITLE
[$10][MSFT-55] [Register] user-scalable=no on <meta> tag disables zooming on mobile devices

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2772]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Open the source and check

**Expected Result**
Ensures <meta name="viewport"> does not disable text scaling and zooming

**Actual Result**
user-scalable=no on <meta> tag disables zooming on mobile devices

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.4 Resize Text
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F31

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096298 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096298 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sumitdaga** [comment]
Please assign this to me!

**Comment by veshu** [comment]
@sumitdaga Can you please register here https://www.topcoder.com/challenges/30095935/?type=develop&nocache=true&tab=details and use this link to register https://x.topcoder.com/api/v1/github/teams/registration/462d863d-9f1a-4179-a215-29f71a551e4f-1563464121672

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096298 has been updated - it has been assigned to daga_sumit.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sumitdaga** [comment]
@veshu  
i have done both, but i still need to be added to some group i guess, i cannot modify labels on issues, etc  
thanks

**Comment by sumitdaga** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/231

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2772 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2772 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2772 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2772 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097502 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097502 has been updated - it has been assigned to daga_sumit.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows/Android
Browser: Chrome
Page: https://accounts.topcoder.com/member

**Comment by drasticdpk** [comment]
Operating System: Windows/Android
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097502<br/><br/>```This is an automated message for crazyk via Topcoder X```
